diff --git a/src/simd_instruction.h b/src/simd_instruction.h
index 41deb16..14510b3 100644
--- a/src/simd_instruction.h
+++ b/src/simd_instruction.h
@@ -24,12 +24,12 @@
 #include "simde/simde/x86/avx512.h"
 #else
 #ifdef __AVX2__
-#include "simde/simde/x86/avx2.h"
+#include "simde/x86/avx2.h"
 #else
 #ifdef __SSE4_1__
 #include "simde/simde/x86/sse4.1.h"
 #else
-#include "simde/simde/x86/sse2.h"
+#include "simde/x86/sse2.h"
 #endif // end of sse41
 #endif // end of AVX2
 #endif // end of 512F
